src/pages/teams.astro

```diff
@@ -133,6 +134,27 @@ const future = false;
             and copy-pasting them to Livebook.</span
           >
         </div>
+        <div class="m-auto max-w-2xl flex flex-col items-center">
+          <img
+            class="sm:max-w-sm"
+            src={featureFilesystem}
+            alt="Feature: Synchronized file storage access"
+            width="571"
+            height="247"
+          />
+          <h2
+            class="text-gray-50 text-2xl md:text-3xl leading-tight mt-8 sm:mt-10"
+          >
+            Synchronized file storage access
+          </h2>
+          <span class="text-gray-200 mt-6 sm:mt-8"
+            >Share meaningful data and resources across all team members.
+            Livebook Teams automatically synchronizes your cloud storage
+            credentials (such as Amazon S3, Google Cloud, etc) across team members,
+            making it easier than ever to write and share notebooks that automate
+            and build insights on data.</span
```

Comment:
@hugobarauna , I gave a first pass. Feel free to do your own and merge when ready. :)

Reply:
@josevalim Will do it now.